docs/admin/federation/index.md

```diff
@@ -0,0 +1,111 @@
+---
+---
+This guide explains how to set up a federation cluster. Federation is an alpha feature which lets you control multiple kubernetes clusters.
+[Federation proposal](https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/federation.md) has more details.
+
+
+* TOC
+{:toc}
+
+## Prerequisites
+
+This guide assumes that you have a running kubernetes cluster.
+If not, then you can head over to the [getting started guides](/docs/getting-started-guides/) to bring up a cluster.
+
+This guide also assumes that you have the kubernetes source code.
+If not, then you can head over to the [download kubernetes guide](/docs/getting-started-guides/binary_release/) to get the source code.
+
+## Setting up federation control plane
+
+### Building and pushing federation control binaries
+
+You can build the federation control plane binaries using the following command:
+
+```shell
+$ FEDERATION=true KUBE_RELEASE_RUN_TESTS=n make quick-release
+```
+
+Next, you need to push the images to a registry from where your kubernetes cluster can read them.
+If your kubernetes cluster is running on GCE, then you can push the images to `gcr.io/<your-gce-project-name>`.
```

Comment:
In general, on this page, I think you can just say cluster; "kubernetes cluster" is not necessary.